app/main.development.js

```diff
@@ -52,8 +53,8 @@ app.on('ready', async () => {
   mainWindow.loadURL(`file://${__dirname}/app.html`);
 
   mainWindow.webContents.on('did-finish-load', () => {
-    mainWindow.show();
-    mainWindow.focus();
+    mainWindow.show(); // eslint-disable-line
```

Comment:
But I think it would not be worth adding that code just to circumvent the eslint error in this case - what do you think @amilajack ?